src/sentry/rules/processing/processor.py

```diff
@@ -236,8 +245,9 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
                 if not predicate_func(predicate_iter):
                     return
             else:
-                self.logger.error(
-                    f"Unsupported {name}_match {match!r} for rule {rule.id}",
+                log_string = f"Unsupported {name}_match {match!r} for rule {rule.id}"
```

Comment:
does this need to be in a separate var?

Reply:
I got some error if i didn't: `LOG011 avoid pre-formatting log messages`